modules/nw-ovn-kubernetes-ansible-migration-about.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-ansible-migration-about_{context}"]
+= Using the Ansible plybook to migrate to the OVN-Kubernetes network plugin
+
+As a cluster administrator, you can use an Ansible playbook to migrate from the OpenShift SDN Container Network Interface (CNI) network plugin to the the OVN-Kubernetes plugin for your cluster.
+
+The Ansible playbook includes the following set of playbooks that each represent a stage in the migration process:
+
+. Prechecks
+. Migrate
+. Reboot
+. Post-migration
+
+[IMPORTANT]
+====
+If you run a playbook as a non-cluster administrator, the playbook fails.
+====
+
+If a playbook fails, check the playbook output for the source of the error. Resolve the error and then rerun the Ansible playbook.
+
+.Prerequisites
+
+* You set a bastion server.
+// LINK for this packages in a RH resource?
+* You installed `ansible`, `jq`, `pip3`, and `python3` packages.
+** To install the `ansible` package, enter the `python3 -m pip install --user ansible` command.
+// SET TO WHAT?
+* You set the `KUBECONFIG` environment variable.
+* See "Migrating to the OVN-Kubernetes network plugin by using the offline migration method" for a list of other prerequisites.
+
+.Procedure
+
+// Is this a safe resource
+. Clone the link:https://github.com/openshift/ansible-ocp-networking-migration-rollback[`ansible-sdn-to-ovn-migration.git`] repository to your local system.
```

Comment:
Once the playbooks are published in ansible automation hub, this step would change.

Reply:
OK. For now, we have no choice but to use a GH resource. When will the playbooks be published to the Ansible automation hub?

Reply:
Right now, we do not have an ETA. @miheer will update as soon as we have the info.